### DISCUSSION TITLE
Delete map on the top (I don't know what is the real name, I'm noob, sorry)

### DISCUSSION THREAD

**Discussion by Agus9217** [LazyVim/LazyVim#2996] [Q&A] [answered]
<img width="1113" alt="Captura de Pantalla 2024-04-15 a la(s) 23 56 00" src="https://github.com/LazyVim/LazyVim/assets/86172739/f1e79bce-eb3d-4f9e-91af-1b3469d8a3d9">

I really don't like this configuration, i want to delete all brackground gray, can anyone help me? I'm noob, and a don't know what is the really name thanks, srry for my english

**Comment by dpetka2001** [comment, marked as answer]
[This](https://www.lazyvim.org/plugins/treesitter#nvim-treesitter-context) is the plugin that causes this behavior and you can disable it as per the [docs](https://www.lazyvim.org/configuration/plugins#-disabling-plugins).

**Reply by Agus9217** [reply]
Thanks a lot
